### ISSUE TITLE
Objectives

### ISSUE THREAD

**Issue by publishwhatyoufund** [pwyf/2022-Index-indicator-definitions#31]
## Description

The objectives or purposes of the activity are those that the activity intends to achieve.

This is only expected if the activity is in the implementation, completion or post-completion phase.

This is not expected if the default aid type code is administrative costs (G01).

## Proposed test

```python
   Given an IATI activity
    And the activity is current
    And `activity-status/@code` is one of 2, 3 or 4
    And `default-aid-type/@code` is not G01
    Then `document-link/category[@code="A02"] or description[@type="2"]` should be present
```

**Comment by ToonVB** [comment]
Hello, in practice there is no info on (intended nor achieved) results for core contributions either (type of aid B01-B04), where it is the implementer that owns & publishes this info. For small scale type of aid D & H activities (of which bilateral donors have a lot of ad hoc contributions) this info is regularly lacking as well. Finally, the same goes for type of aid F financing arrangements.
In short , as is the case with a number of other info-elements, this is only collected/available in a systematical way for type of aid C1 activities = specific projects.

**Comment by publishwhatyoufund** [comment]
**@ToonVB** We have not excluded these aid types from the performance indicators since core contributions can still have objectives and results.
